### PR TITLE
[zh-cn] Refresh kubeadm docs for Windows

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Signed-off-by: Guangwen Feng <fenggw-fnst@fujitsu.com>

en PR #33324

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md [hunk 1]
```diff
@@ -111,26 +111,52 @@ upgrade the control plane nodes before upgrading your Windows nodes.
...
+    restart-service kube-proxy
+    ```
+-->
+2. 在 Windows 节点上升级并重启 kube-proxy：
+
+    ```powershell
+    stop-service kube-proxy
+    curl.exe -Lo <path-to-kube-proxy.exe> https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kube-proxy.exe
+    restart-service kube-proxy
+    ```
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh-cn/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md]: Thanks for the review, done.

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md [hunk 2]
```diff
@@ -111,26 +111,52 @@ upgrade the control plane nodes before upgrading your Windows nodes.
...
 -->
-### 升级 kubelet   {#upgrade-kubelet}
+### 升级 kubelet 和 kube-proxy   {#upgrade-kubelet-and-kube-proxy}
 
 1. 在 Windows 节点上升级并重启 kubelet：
 
    ```powershell
    stop-service kubelet
-   curl.exe -Lo C:\k\kubelet.exe https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kubelet.exe
+   curl.exe -Lo <path-to-kubelet.exe> https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kubelet.exe
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh-cn/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md]: Thanks, done.
Also modified the other two <path-to-xxx> to <xxx  路径>.

File: content/zh-cn/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md [hunk 3]
```diff
@@ -111,26 +111,52 @@ upgrade the control plane nodes before upgrading your Windows nodes.
...
+    ```
+
+<!--
+If you are running kube-proxy in a HostProcess container within a Pod, and not as a Windows Service, you can upgrade kube-proxy by applying a newer version of your kube-proxy manifests.
+-->
+{{< note >}}
+如果你是在 Pod 内的 HostProcess 容器中运行 kube-proxy，
+而不是作为 Windows 服务，你可以通过应用更新版本的
+kube-proxy 清单文件来升级 kube-proxy。
+{{< /note >}}
```
[the remark this answers is not in the document]
Comment (fenggw-fnst) [reply content/zh-cn/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md]: Thanks, updated as suggested.

**Comment by tengqm** [comment]
This is clean. Thanks.
Next time, you may want to mark the feedbacks as resolved after revised version is committed.

/approve

**Pull request merged**